### PR TITLE
Skip disk eviction test on non-supported images.

### PR DESCRIPTION
**Pull request by ronnielai**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31200)

### CODE REVIEW

File: test/e2e_node/disk_eviction_test.go [hunk 1]
```diff
@@ -217,3 +221,9 @@ func recordContainerId(containersToCleanUp map[string]bool, containerStatuses []
 func evictionOptionIsSet() bool {
 	return len(framework.TestContext.EvictionHard) > 0
 }
+
+func isImageSupported() bool {
+	// TODO: Only images with image fs is selected for testing for now. When the kubelet settings can be dynamically updated,
+	// instead of skipping images the eviction thresholds should be adjusted based on the images.
+	return strings.Contains(framework.TestContext.NodeName, "-gci-dev-")
+}
```
Comment (Random-Liu) [inline test/e2e_node/disk_eviction_test.go]: Wired sign here. Maybe a new line here?

**Comment by Random-Liu** [comment]
LGTM with one small nit.

File: test/e2e_node/disk_eviction_test.go (hunk 1 above)
Comment (ronnielai) [inline test/e2e_node/disk_eviction_test.go]: done

**Comment by ronnielai** [comment]
Applied LGTM after addressing the nit.

**Comment by derekwaynecarr** [comment]
lgtm too.

**Comment by dchen1107** [comment]
Raised the priority to P1 since it breaks node e2e completely. Thanks!

**Pull request merged**